release-please-config.json

```diff
@@ -1,5 +1,9 @@
 {
-  "separate-pull-requests": true,
+  "bootstrap-sha": "141faf0c19eae382d0e19833129f5545fc2355c8",
```

Comment:
This is a one time config value needed to tell `release-please` how far back to look for the previous release. It can be deleted after this PR is merged.